clarifai/client/auth/helper.py

```diff
@@ -5,11 +5,15 @@
 
 from clarifai_grpc.channel.clarifai_channel import ClarifaiChannel
 from clarifai_grpc.grpc.api import resources_pb2, service_pb2_grpc
+
+from clarifai import __version__
 from clarifai.utils.constants import CLARIFAI_PAT_ENV_VAR, CLARIFAI_SESSION_TOKEN_ENV_VAR
 
 DEFAULT_BASE = "https://api.clarifai.com"
 DEFAULT_UI = "https://clarifai.com"
 
+REQUEST_ID_PREFIX = "sdk-python-" + str(__version__)
```

Comment:
How about GRPC case? or is that a different repo?

Reply:
Yes